storage/elasticsearch/client.go

```diff
@@ -156,13 +156,25 @@ func (c *Client) addMapping(index Index) error {
 	return nil
 }
 
+func (c *Client) putSetting(index Index) error {
+	const setting string = `{ "index.mapping.total_fields.limit": 10000 }`
```

Comment:
I would prefer to know and to be sure about what we are going to store with a limited amount of fields. I think we need to handle this in the probes.

Reply:
@safchain - alternative fixes could be:

1. arbitrarily limit the number of fields in k8s (as we have many types of resources we are handling its' not always clear which subfield will overflow and cause us to exceed ElasticSearch max field limit.
2. serialize the entire k8s field as a BLOB (for all backends) and then after reading from the backend and before transferring to UI we translate into a multi-level map which can be displayed correctly in the UI.

Let me know what you think about these options

Reply:
@hunchback maybe we could do both, limiting the number of field to well known fields so that we can continue to do gremlin filter on them and if more is needed we can use blob. 

I would first limit the fields and then adding the blob thing. Maybe that is something we could extend (thinking about lazy loading, other es index)